src/armature/Node.ts

```diff
@@ -6,34 +6,56 @@ import { matrix4, vector3 } from '../types/VectorTypes';
 import { NodeRenderObject } from './NodeRenderObject';
 import { Transformation } from './Transformation';
 
+import { flatten, flatMap } from 'lodash';
+
 /**
  * A `Node` in a scene-graph.
  */
 export class Node {
-    private static boneVertices: vec3[] = [
-        vec3.fromValues(0, 0, 0),
-        vec3.fromValues(0.5, 0.1, 0),
-        vec3.fromValues(0.5, 0, -0.1),
-        vec3.fromValues(0.5, -0.1, 0),
-        vec3.fromValues(0.5, 0, 0.1),
-        vec3.fromValues(1, 0, 0)
+    private static boneVertices: number[][] = [
+        [0, 0, 0],
+        [0.5, 0.1, 0],
+        [0.5, 0, -0.1],
+        [0.5, -0.1, 0],
+        [0.5, 0, 0.1],
+        [1, 0, 0]
     ];
 
     private static bone: BakedGeometry = {
-        vertices: Node.boneVertices,
-        normals: [
-            vec3.fromValues(-1, 0, 0),
-            vec3.fromValues(0, 1, 0),
-            vec3.fromValues(0, 0, -1),
-            vec3.fromValues(0, -1, 0),
-            vec3.fromValues(0, 0, 1),
-            vec3.fromValues(1, 0, 0)
-        ],
-        indices: [0, 1, 2, 0, 2, 3, 0, 3, 4, 0, 4, 1, 5, 2, 1, 5, 3, 2, 5, 4, 3, 5, 1, 4],
+        vertices: Float32Array.from(flatten(Node.boneVertices)),
+        normals: Float32Array.from(
+            flatten([[-1, 0, 0], [0, 1, 0], [0, 0, -1], [0, -1, 0], [0, 0, 1], [1, 0, 0]])
+        ),
+        indices: Int16Array.from([
```

Comment:
nit: can you change this to be on a single line again?

Reply:
the autoformatter does this, I think the line is too long. I can group them in threes and flatten like above, if that's better?

Reply:
Yeah, that would be good